strategy/inuse.go

```diff
@@ -110,6 +116,43 @@ func (strat *inUseStragey) Run(policy v1alpha1.SnapshotPolicy) (*v1alpha1.Snapsh
 	return retStatus, retErr
 }
 
+func (strat *inUseStragey) getPvcNames(policy v1alpha1.SnapshotPolicy) ([]string, error) {
+	pvcNames := make(map[string]bool)
+	if policy.Spec.PVCNames != nil {
+		for _, pvcName := range *policy.Spec.PVCNames {
+			pvcNames[pvcName] = true
+		}
+	}
+	errors := make([]string, 0)
```

Comment:
```suggestion
	var errors []string
```
as errors wont be allocated if it is unused.

Reply:
fixed, thanks